mkdocs.yml

```diff
@@ -0,0 +1,69 @@
+---
+nav:
+    - Home: index.md
+    - Installation: installation/index.md
+    - Getting Started:
+          - 1. Initial Steps: getting-started/initial-steps.md
+          - 2. Create a Component: getting-started/create-component.md
+          - 3. Use the Template Tag: getting-started/reference-component.md
+          - 4. Render Your View: getting-started/render-view.md
+          - 5. Learn More: getting-started/learn-more.md
+    - Exclusive Features:
+          - Hooks: features/hooks.md
+          - Template Tag: features/templatetag.md
+    - Contribute:
+          - Django-IDOM Environment: contribute/idom-env.md
+          - Documentation Environment: contribute/docs-env.md
```

Comment:
I think we could remove the "Environment" part of the section title. As is, I read this as "How to _Contribute_ to the _Django-IDOM Environment_" rather than just how to contribute to Django-IDOM.

Reply:
I've renamed the two sections to `code`, and `docs` respectively. Let me know what you think.